runtime/src/epoch_stakes.rs

```diff
@@ -101,20 +101,6 @@ impl EpochStakes {
             .iter()
             .filter_map(|(key, (stake, account))| {
                 let vote_state = account.vote_state();
-                let vote_state = match vote_state.as_ref() {
```

Comment:
A few lines above we would have accumulated `total_stake` for invalid vote accounts if they were in the epoch vote accounts map. Now invalid vote accounts can't be in the epoch vote accounts map.